ddtrace/contrib/dbapi/__init__.py

```diff
@@ -3,33 +3,19 @@
 """
 import six
 
-from ddtrace import config
-from ddtrace.vendor import debtcollector
-
 from ...constants import ANALYTICS_SAMPLE_RATE_KEY
 from ...constants import SPAN_MEASURED_KEY
 from ...ext import SpanTypes
 from ...ext import sql
 from ...internal.logger import get_logger
 from ...pin import Pin
-from ...utils.attrdict import AttrDict
-from ...utils.formats import asbool
-from ...utils.formats import get_env
 from ...vendor import wrapt
 from ..trace_utils import ext_service
 from ..trace_utils import iswrapped
 
 
 log = get_logger(__name__)
 
-config._add(
```

Comment:
@majorgreys should this config stay and keep only the `_default_service` setting?

Reply:
Good question. We had deprecated warnings for when a user used the `trace_fetch_methods` or the analytics sample rates for dbapi. But we missed the need to also add a deprecation warning for if the `config.dbapi2` was used directly for setting the service name. That seems like a tricky case to catch.

Is it necessary to remove this config object as part of the deprecation? If someone is using it now for their own custom integration, then they could continue doing so, but would have to pass it into when initializing the cursor.

Reply:
> Is it necessary to remove this config object as part of the deprecation?

No I don't think we need to remove it and this is why I was asking. I believe some of the tests are failing because I've removed too much. Just wanted to get your thoughts on this.